digitalpublicgoods/pharmacovigilance-electronic-reporting-tool.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "Pharmacovigilance Electronic Reporting Tool",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://pv.pharmacyboardkenya.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.google.com/document/d/1LjkPOKrK-4O_PBb4GRfx61I2_0zfesD_5X1y9aw34dQ/edit#heading=h.6nlwssopwwv3"
```

Comment:
The documentation should not be password protected. Please request the submitter to make the latter publicly available. Thanks.

Reply:
Resolved in commit fcc426e